src/system/RenderableSystem.lua

```diff
@@ -20,6 +20,27 @@ function RenderableSystem:setScene ( tag )
 
 end
 
+function RenderableSystem:update( dt )
```

Comment:
Another consideration is that we might want to have the same game element play different animations (like, an animation for when a space cruiser nukes a planet from orbit). In this case, maybe a Renderable should be able to receive an Animation to play once/on-repeat. The Renderable's 'sprite' would just be a default to display when it wasn't showing an animation and had no idle-animation.

I'll admit I've not done a game with 2d animations before, so not sure what the industry-standard approach is. ;)